buildpipeline/windows.groovy

```diff
@@ -53,6 +53,7 @@ simpleNode('windows.10.amd64.clientrs4.devex.open') {
             bat ".\\build-tests.cmd ${framework} -buildArch=${params.AGroup} -${params.CGroup}${additionalArgs} -- /p:RuntimeOS=win10 /p:ArchiveTests=${archiveTests} /p:EnableDumpling=true"
         }
         else {
+            bat ".\\build-tests.cmd -framework:netstandard -buildArch=${params.AGroup} -${params.CGroup} -SkipTests"
```

Comment:
Do we plan to also add this to our official builds?

Reply:
we could, but we don't have to. The purpose of adding it here is so that the netstandard vertical of the tests won't get broken by check-ins. The actual netstandard test suite doesn't really need to be signed in order for people to consume it (like Xamarin) so they can even build the test suite themselves. 

I know that the all configurations leg is the long pole on our official builds, so I wouldn't want to add something that will make it longer if it is not required. That said, if you want me to add it, I can certainly do it.

Reply:
No I wouldn't add it to the official build legs unless we plan to produce a drop of the tests for folks to consume. If they are going to be building them themselves then what you have is enough.